apps/src/templates/manageStudents/ManageStudentsTable.jsx

```diff
@@ -729,6 +747,31 @@ class ManageStudentsTable extends Component {
               )}
             </div>
           )}
+          {noSectionCode.includes(loginType) && (
+            <div style={styles.sectionCodeBox}>
+              {i18n.sectionCodeWithColon()}
+              <strong>
```

Comment:
Feel free to ignore if this was known/intentional. `<strong>` is generally used to indicate important warning text. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong. This feels more informational. Would the css `font-weight` property be better?

Reply:
That’s a good point – I wouldn’t say it is important warning text, so the `<strong>` tag is likely not appropriate. I am having an issue with `fontWeight` though. For some reason, setting `fontWeight` to ‘bold’ is not changing the styling (even when I use inline styling). From what I’ve researched, it seems like some font sizes are too small to display bold with fontWeight.

Reply:
I took a look to see if there was a way we could work around this. It looks like `fontWeight='bold'` is just not very bold for this font. Even at max value of 1000, it doesn't change the text very much. `sectionCode` in this component seems to solve for this by changing the font-family to `"Gotham 7r", sans-serif`. 

Another thought. Some of the styling is different from other nearby `<a>` elements. Some suggestions:
```
color: color.teal
textDecoration: 'none'
cursor: 'pointer'
```